src/commonMain/kotlin/baaahs/shows/ThumpShow.kt

```diff
@@ -38,22 +41,24 @@ object ThumpShow : Show("Thump") {
         }
 
         override fun nextFrame() {
-            val theta = ((getTimeMillis() / 1000f) % (2 * PI)).toFloat()
-            val beat = beatProvider.beat
+//            val theta = ((getTimeMillis() / 1000f) % (2 * PI)).toFloat()
```

Comment:
Did you want to leave `theta` out?

Reply:
I have no idea what theta is in this context, what's it do?

Reply:
I think it was just more timey-wimey inputty stuff, dunno. But maybe keep it or kill it rather than commenting it?